index.src.html

```diff
@@ -796,15 +809,17 @@ <h3 id=interface-reporting-observer>Interface {{ReportingObserver}}</h3>
   A <dfn id=dom-report interface>Report</dfn> is the application exposed
   representation of a <a>report</a>. <dfn attribute for="Report">type</dfn>
   returns [=report/type=], <dfn attribute for="Report">url</dfn> returns
-  [=report/url=] and <dfn attribute for="Report">body</dfn> returns
+  [=report/url=], and <dfn attribute for="Report">body</dfn> returns
   [=report/body=].
     
   Each {{ReportingObserver}} object has these associated concepts:
-    - A <dfn for=ReportingObserver>callback</dfn> set on creation.
-    - An {{ReportingObserverOptions}} dictionary called
+    - A <dfn for=ReportingObserver>callback</dfn> function set on creation.
+    - A {{ReportingObserverOptions}} dictionary called
       <dfn for=ReportingObserver>options</dfn>.
     - A list of {{Report}} objects called the <dfn for=ReportingObserver>report
-      list</dfn> that is initially empty.
+      list</dfn>, which is initially empty if the <a>context object</a> has a
+      {{futureOnly}} <a>option</a> that is true, or a copy of the <a>report
```

Comment:
I think the preferred style is to describe non-trivial initialization like this in terms of an algorithm.  I.e. think of this section as your variable declarations.  It doesn't make sense to say "initially a copy of the report buffer .. if the context object has a futureOnly option" because *initially* the ReportingObserver has no options.  I think you want to instead update the algorithm for the `observe` method to say something like "if the futureOnly option is false, set the report list to a copy of the report buffer" etc.  It's probably important to define exactly when this happens because it may be observable (which reports are present).

Reply:
I have added this to the algorithm for <code>observe</code>.